roles/keycloak/defaults/main.yaml

```diff
@@ -0,0 +1,11 @@
+# Default variables for Dojot's Keycloak
+
+dojot_keycloak_volume_size: 2Gi
```

Comment:
Leo, 2Gi não é pouco, ou é só um valor inicial ?

Reply:
O valor de 2Gi não foi um valor "calculado", mas sim algo inicial mesmo. Podemos fazer alguns testes e aumentar no futuro, se for o caso!

Reply:
Perfeito!!